packages/test/test-end-to-end-tests/src/test/summaries.spec.ts

```diff
@@ -402,6 +411,99 @@ describeCompat("Summaries", "NoCompat", (getTestObjectProvider) => {
 		await provider.ensureSynchronized();
 		await summaryCollection.waitSummaryAck(container.deltaManager.lastSequenceNumber);
 	});
+
+	async function getNackProm(container: IContainer) {
```

Comment:
nit: I haven't seen `Prom` anywhere else in the repo, probably best to stick with `Promise` or `P`